ui/src/main/java/org/pentaho/di/ui/core/database/wizard/CreateDatabaseWizard.java

```diff
@@ -71,8 +71,6 @@ public DatabaseMeta createAndRunDatabaseWizard( Shell shell, PropsUI props, List
 
     final CreateDatabaseWizardPageOCI pageoci = new CreateDatabaseWizardPageOCI( "oci", props, newDBInfo );
 
-    final CreateDatabaseWizardPageODBC pageodbc = new CreateDatabaseWizardPageODBC( "odbc", props, newDBInfo );
```

Comment:
This 'CreateDatabaseWizardPageODBC' class should also be removed from code!
It exists here: [ui/src/main/java/org/pentaho/di/ui/core/database/wizard](https://github.com/pentaho/pentaho-kettle/tree/master/ui/src/main/java/org/pentaho/di/ui/core/database/wizard)

Don't forget to also remove the messages referring to this same class - check message files here: [ui/src/main/resources/org/pentaho/di/ui/core/database/wizard/messages](https://github.com/pentaho/pentaho-kettle/tree/master/ui/src/main/resources/org/pentaho/di/ui/core/database/wizard/messages)